lib/edge_impulse/Kconfig

```diff
@@ -6,9 +6,9 @@
 
 menuconfig EDGE_IMPULSE
 	bool "Enable Edge Impulse"
-	depends on CPLUSPLUS
+	depends on CPP
 	depends on STD_CPP11
-	depends on LIB_CPLUSPLUS
+	depends on EXTERNAL_LIBCPP
```

Comment:
The `CONFIG_LIB_CPLUSPLUS` selects `CONFIG_REQUIRES_FULL_LIBCPP` (which would enable `CONFIG_GLIBCXX_LIBCPP`). I wonder if we can use `CONFIG_EXTERNAL_LIBCPP` instead here (is the external C++ standard library actually provided in our case)?
Seemingly everything builds without it so consider double checking with Edge Impulse team if it's still needed (keep in mind that it may be used e.g. only by subset of machine learning algorithms).